src/core/ItemContainer.tsx

```diff
@@ -0,0 +1,10 @@
+import * as React from "react";
+import { ComponentCompat } from "../utils/ComponentCompat";
+
+export interface ItemContainerProps {
```

Comment:
call it content container

Reply:
This file has been removed as it was not necessary